manifests/l/LWorks/SuperEggHunt/1.53/LWorks.SuperEggHunt.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: LWorks.SuperEggHunt
+PackageVersion: 1.53
+Installers:
+- Architecture: neutral
+  InstallerType: inno
+  InstallerUrl: https://lworksus.s3.us-east-1.amazonaws.com/superegghuntsetup.exe
+  InstallerSha256: 5AAFD9E02DAE909C65C1CF921B19511EDB74A81C5839D51CC8557EBF0F66FA60
+  InstallerSwitches:
+    Silent: /verysilent

```

AppsAndFeaturesEntries will help with product matching in the future since display version is empty
```suggestion
    Silent: /verysilent
    AppsAndFeaturesEntries: 
    - DisplayName: Super Egg Hunt 1.6
      DisplayVersion: null
      Publisher: LWorks
      ProductCode: Super Egg Hunt_is1
```